libcxx/test/libcxx/diagnostics/unordered_map.nodiscard.verify.cpp

```diff
@@ -6,20 +6,104 @@
 //
 //===----------------------------------------------------------------------===//
 
-// UNSUPPORTED: c++03
+// REQUIRES: std-at-least-c++11
```

Comment:
I think we should remove this as we backported unordered associative containers to C++03 mode. Same for tests for `unordered_multimap`, `unordered_set`, and `unordered_multiset`.

Reply:
OK. I suggest to do this refactoring in a single commit separately after these land to avoid quadruple CI run if that's the only issue? WDYT?